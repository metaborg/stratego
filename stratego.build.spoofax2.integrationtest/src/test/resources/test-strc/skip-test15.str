module test15

imports libstratego-lib

strategies
  main : a -> a with 1
