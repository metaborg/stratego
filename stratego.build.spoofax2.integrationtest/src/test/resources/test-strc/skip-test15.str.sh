#!/bin/sh

java -jar ${STRJ} -i test15.str -la stratego-lib || exit 1 

rm -f test15.str.actual

javac -cp ${STRJ} test15.java || exit 1

java -cp .:${STRJ} -Xmx1024m test15 > test15.str.actual 2>&1

diff test15.str.expected test15.str.actual || (echo "Result not expected; test failed"; exit 1)
