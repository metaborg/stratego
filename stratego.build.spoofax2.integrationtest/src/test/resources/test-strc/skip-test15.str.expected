Internal error: 'with' clause failed unexpectedly in 'main'
	"test15"
	main_0_0
	with_1_1 <==
